style.css

```diff
@@ -115,7 +115,39 @@ a:hover {
 /* MAIN CONTENT */
```

Comment:
## Thumbs up 👍 
Great work here on the stylesheet. The rules are neatly defined. Good job. ✔️